conditions/src/main/scala/explore/conditions/Conditions.scala

```diff
@@ -7,17 +7,32 @@ import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
 import react.semanticui.elements.button._
 import react.semanticui.colors._
+// import crystal.react.io.implicits._
+import explore.model._
+import scala.scalajs.js
+import js.UndefOr._
+import cats.effect._
 
 object Conditions {
 
+  @inline implicit def io2UndefOrCallback[A](io: IO[A]): js.UndefOr[Callback] = Callback {
+    io.unsafeRunAsyncAndForget()
```

Comment:
Isn't this part of your PR to `scalajs-react`?

Reply:
Not necessarily, the PR to `scalajs-react` doesn't deal with `IO`.

This code won't stay here, we have to move it to some common place. There's similar code when you `import crystal.react.io.implicits._` to implicitly convert from `IO` to `Callback`, but in this case `Button` expects a `js.UndefOf[Callback]` and so implicit conversion was not kicking in. (Scala.js has another implicit to go from `A -> js.UndefOr[A]` but it doesn't chain.

I will move this functionality to `crystal.react.io.implicits` with the rest of the implicit conversions.